services/brig/test/integration/API/User/Connection.hs

```diff
@@ -489,6 +491,41 @@ testBlockAndResendConnection brig galley = do
   getConversation galley uid1 cnv !!! do
     const 200 === statusCode
 
+data ConnectRequestReaction = Ignore | Block
+
+testResendUnblockedConnection :: ConnectRequestReaction -> Opt.Opts -> Brig -> Http ()
+testResendUnblockedConnection reqReact opts brig = do
+  let throttleCount :: Int = 3 -- times (default value for prod: 3 times)
+      throttleTimeWindow :: Int = 3 -- seconds (default value for prod: 7 days)
+      newOpts = error "TODO" opts throttleCount throttleTimeWindow
+  u1 <- randomUser brig
+  u2 <- randomUser brig
+  let uid1 = userId u1
+  let uid2 = userId u2
+
+  -- TODO: i'm pretty sure this is not how `withSettingsOverrides` works, but it's used like
+  -- this all over the place?!  we shouldn't pass in an (unaltered) brig that we're then
+  -- calling, but we should use the application from the `Session` context to run the requests
+  -- again.  need to investigate!
+
+  withSettingsOverrides newOpts $ do
+    -- Initiate a new connection (A -> B)
+    postConnection brig uid1 uid2 !!! const 201 === statusCode
+    -- User B reacts
+    case reqReact of
+      Ignore -> pure ()
```

Comment:
```suggestion
      Ignore -> putConnection brig uid2 uid1 Ignored !!! const 200 === statusCode
```

...  or something like that.  ignoring is a pushing a button, not the absence of doing anything.